### PR TITLE
feat: Implement Rewards Banner warning behind env var.

### PR DESCRIPTION
**Pull request by Tulun**
Screen Shot 2022-07-28 at 12 22 45 PM

Banner hidden behind REACT_APP_REWARDS_BANNER_WARNING = true. Only shows on /rewards page.

### CODE REVIEW

File: src/components/Banner/Banner.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    if (!container.current) {
+      // we know this to always be defined.
+      const root = document.getElementById("root") as HTMLDivElement;
+      const div = document.createElement("div");
+      div.id = "banner";
+      root.insertBefore(div, root.firstChild);
+      container.current = div;
+    }
+  }, []);
+  if (!container.current) {
```
Comment (nicholaspai) [inline src/components/Banner/Banner.tsx:20]: Out of curiosity, does it increase performance to exit earlier if !container above where it's first defined?

**Review by nicholaspai — APPROVED** [review]
LGTM whenever we want to start displaying the warning

File: src/components/Banner/Banner.tsx (hunk 1 above)
Comment (Tulun) [reply src/components/Banner/Banner.tsx:20]: Hm it might, not sure if it would be a huge diff.

**Comment by daywiss** [comment]
@nicholaspai @Tulun i've added the env to enable the banner just for preview and development

**Comment by daywiss** [comment]
looks good, but just fyi the banner at the top extends the length of the page causing a scrollbar to show up. this was the same issue with the navbar that was fixed recently.  you could fix in this pr or make an issue for it, up to you

**Review by daywiss — APPROVED** [review]
only issue is that banner causes page length to be beyond window height, you can fix in this pr or make an issue for it since its pretty minor

File: src/components/Banner/Banner.styles.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    padding: 0 10px;
+  }
+  svg {
+    margin-right: 16px;
+  }
+
+  span {
+    padding: 10px 0;
+    @media ${QUERIES.tabletAndDown} {
+      font-size: ${13 / 16}rem;
```
Comment (amateima) [inline src/components/Banner/Banner.styles.tsx:23]: this should be 14, not 13

File: src/components/Banner/Banner.styles.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    }
+    @media ${QUERIES.mobileAndDown} {
```
Comment (amateima) [inline src/components/Banner/Banner.styles.tsx:25]: let's use the new breakpoints from the design. From what I could see in the design, the styles are changed only on mobile ( <= 428px )

**Review by amateima — CHANGES_REQUESTED** [review]

File: src/components/Banner/Banner.styles.tsx (hunk 3 above)
Comment (Tulun) [reply src/components/Banner/Banner.styles.tsx]: I've changed this but it won't look good at all breakpoints.

**Review by amateima — APPROVED** [review]

**Pull request merged**